### PR TITLE
Added favorites.

### PR DESCRIPTION
**Pull request by jpmensching**

### CODE REVIEW

File: src/js/favorites.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	});
+}
+
+function sync() {
+	if (localStorage) {
+		try {
+			localStorage.setItem(lsKey, JSON.stringify(favorites));
+		} catch (e) {
+			console.error(e);
+			alert('We\'re sorry, your browser does not support this feature.');
```
Comment (luqmaan) [inline src/js/favorites.js]: This is probably a bad thing to do. :dancer:

File: src/js/favorites.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+favoritesHelper.isFavorite = function(id) {
+	return favorites.indexOf(id) > -1;
+};
+
+favoritesHelper.subscribe = function(_listener) {
+	listeners.push(_listener);
+};
+
+module.exports = favoritesHelper;
```
Comment (luqmaan) [inline src/js/favorites.js]: Need a newline here.

**Comment by luqmaan** [comment]
This works great! Thank you so much.

:bus: 

I'm going to merge this as is.

**Comment by jpmensching** [comment]
Thanks! I'm gonna remove that alert (stupid idea) and add the missing newline in another PR.

**Pull request merged**